src/test/java/de/adorsys/keycloak/config/AbstractImportTest.java

```diff
@@ -65,7 +65,7 @@
                 .withExposedPorts(8080)
                 .withEnv("KEYCLOAK_USER", "admin")
                 .withEnv("KEYCLOAK_PASSWORD", "admin123")
-                .withEnv("KEYCLOAK_LOGLEVEL", "WARN")
+                .withEnv("KEYCLOAK_LOGLEVEL", "INFO")
```

Comment:
Any reason to be change this?

Reply:
Only for debugging purpose, WARN is not enough. 
I believe that could help the next guy to debug.
I can revert this if you want.

Reply:
All right. Since the runtime for integration tests are not changed, it's fine.